src/status_im/contexts/wallet/send/select_address/tabs/view.cljs

```diff
@@ -17,15 +17,20 @@
         :description     (i18n/label :t/here-is-a-cat-in-a-box-instead)
         :image           (resources/get-themed-image :cat-in-box theme)
         :container-style style/empty-container-style}]
-      (into [rn/view {:style style/my-accounts-container}]
-            (map (fn [{:keys [color address] :as account}]
-                   [quo/account-item
-                    {:account-props (assoc account :customization-color color)
-                     :on-press      #(rf/dispatch [:wallet/select-send-address
-                                                   {:address   address
-                                                    :recipient account
-                                                    :stack-id  :screen/wallet.select-address}])}]))
-            other-accounts))))
+      [rn/view {:style style/my-accounts-container}
+       (for [{:keys [color address] :as account} other-accounts]
+         ^{:key (str address)}
+         (let [transformed-address (rf/sub [:wallet/account-address address
```

Comment:
@mmilad75 The `for` macro produces a lazy sequence and `rf/sub` is actually deref'ing a reaction (atom), which is a side-effect. Lazy sequences with side-effects should then be wrapped by `doall`. This same problem would exist with `map`.

Reply:
Done